### PR TITLE
DSC AEC

### PR DESCRIPTION
**Pull request by subratkishoredutta**
Fixes #

## Proposed Changes

-
-
-

### CODE REVIEW

File: src/components/json/worldData.js [hunk 1]
```diff
@@ -30,6 +30,21 @@ const data = [
 		twitter: "https://twitter.com/abhinavsri360",
 		portfolio: "https://abhinavsri360.netlify.app",
 	},
+	{
+		longitude: 91.666740,
+		latitude: 26.154620,
+		batch: 2020,
+		photo: subrat,
```
Comment (abhinavsri360) [inline src/components/json/worldData.js:37]: this is supposed to be
```
photo: subratkishoredutta
```

**Review by abhinavsri360 — CHANGES_REQUESTED** [review]
Moreover, can you please pull the changes and update the PR?

File: src/components/json/worldData.js [hunk 2]
```diff
@@ -30,6 +30,21 @@ const data = [
 		twitter: "https://twitter.com/abhinavsri360",
 		portfolio: "https://abhinavsri360.netlify.app",
 	},
+	{
+		longitude: 91.666740,
+		latitude: 26.154620,
+		batch: 2020,
+		photo: subratkishoredutta,
+		region: "Asia",
+		skills: ["DL","ML"],
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:39]:
```suggestion
		skills: ["AI/ML"],
```

**Review by aravindvnair99 — CHANGES_REQUESTED** [review]
@subratkishoredutta Only use skills mentioned in the guidelines.

File: src/components/json/worldData.js [hunk 3]
```diff
@@ -30,6 +30,21 @@ const data = [
...
+		photo: subratkishoredutta,
+		region: "Asia",
+		skills: ["AI/ML"],
+		name: "Subrat Kishore Dutta",
+		bio: "AI research enthusiast",
+		url:
+			"https://dsc.community.dev/assam-engineering-college-jalukbari/",
+		chapterName: "Jaypee Institute of Information Technology, Sec - 128",
+		github: "https://github.com/subratkishoredutta",
+		linkedin: "https://www.linkedin.com/in/subrat-kishore-dutta-70a9a316b/",
```
Comment (aravindvnair99) [inline src/components/json/worldData.js:46]:
```suggestion
		linkedin: "https://linkedin.com/in/subrat-kishore-dutta-70a9a316b",
```

**Review by aravindvnair99 — APPROVED** [review]

**Review by abhinavsri360 — APPROVED** [review]
Thanks for adding yourself!

**Comment by subratkishoredutta** [comment]
> Thanks for adding yourself!

thanks for this platform

**Comment by abhinavsri360** [comment]
> > Thanks for adding yourself!
> 
> thanks for this platform

You are Welcome 😄

**Pull request merged**